src/main/kotlin/com/team4099/robot2023/commands/manipulator/IntakeCubeCommand.kt

```diff
@@ -0,0 +1,29 @@
+package com.team4099.robot2022.commands.intake
+
+import com.team4099.robot2023.config.constants.ManipulatorConstants
+import com.team4099.robot2023.subsystems.manipulator.Manipulator
+import edu.wpi.first.wpilibj2.command.CommandBase
+import org.littletonrobotics.junction.Logger
+
+class IntakeCubeCommand(val manipulator: Manipulator) : CommandBase() {
+  init {
+    addRequirements(manipulator)
+  }
+
+  override fun initialize() {
+
+    manipulator.rollerState = ManipulatorConstants.RollerState.CUBE_IN
+  }
+
+  override fun execute() {
+    Logger.getInstance().recordOutput("ActiveCommands/IntakeCobeCommand", true)
+  }
+
+  override fun end(interrupted: Boolean) {
+    manipulator.rollerState = ManipulatorConstants.RollerState.IDLE
+  }
+
+  override fun isFinished(): Boolean {
+    return false
```

Comment:
Is this what we want here? We should be able to have logic here (both for this and the cone intake command) that can detect when we have grabbed a game piece, and end the command there. This will be very useful for chaining this command with other actions for driver assist features